plugins/analytics-provider-segment/src/index.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2022 The Backstage Authors
```

Comment:
I think this should read in all files

```suggestion
 * Copyright 2023 The Janus IDP Authors
```

Reply:
We are picking up some of the code from a closed PR upstream. I am not sure if we should change the original copyright rather add Janus